### PR TITLE
SVD & QR improvements

### PR DESCRIPTION
**Pull request by luitjens**

### CODE REVIEW

File: examples/svd_power.cu [hunk 1]
```diff
@@ -41,19 +41,21 @@ int main([[maybe_unused]] int argc, [[maybe_unused]] char **argv)
 {
   MATX_ENTER_HANDLER();
 
+#if 1
```
Comment (cliffburdick) [inline examples/svd_power.cu:44]: is this needed?

File: include/matx/operators/svd.h [hunk 2]
```diff
@@ -191,7 +192,7 @@ namespace detail {
       using svd_xform_op = bool;
 
       __MATX_INLINE__ std::string str() const { return "svdpi(" + get_type_str(a_) + ")"; }
-      __MATX_INLINE__ SVDBPIOp(const OpA &a, int iterations) : a_(a), iterations_(iterations)
+      __MATX_INLINE__ SVDBPIOp(const OpA &a, int max_iters=10, float tol = 0.0f) : a_(a), max_iters_(max_iters), tol_(tol)
```
Comment (cliffburdick) [inline include/matx/operators/svd.h:195]: I think the default should only be in the user-facing interface since it would get carried down to here

**Review by cliffburdick — APPROVED** [review]

File: examples/svd_power.cu (hunk 1 above)
Comment (luitjens) [reply examples/svd_power.cu:44]: not needed but I use it to toggle between batched and non-batched which has been shown to be useful for debugging/testing.  We can build examples in 20 seconds where as matx_test is 20 minutes.

File: include/matx/operators/svd.h (hunk 2 above)
Comment (luitjens) [reply include/matx/operators/svd.h]: fixed

**Pull request merged**